tests/src/test_cases/test_anim_timeline.c

```diff
@@ -322,4 +322,86 @@ void test_anim_timeline_reverse(void)
     TEST_ASSERT_EQUAL(20, lv_obj_get_x(obj));
 }
 
+
+void anim1_exec_cb(void * var, int32_t v)
+{
+    LV_UNUSED(var);
+    printf("[anim1] %d\n", v);
+}
+void anim1_start(lv_anim_t * a)
+{
+    LV_UNUSED(a);
+    printf("[anim1] start\n");
+}
+void anim1_completed(lv_anim_t * a)
+{
+    LV_UNUSED(a);
+    printf("[anim1] completed\n");
+}
+
+void anim2_exec_cb(void * var, int32_t v)
+{
+    LV_UNUSED(var);
+    printf("                    [anim2] %d\n", v);
+}
+void anim2_start(lv_anim_t * a)
+{
+    LV_UNUSED(a);
+    printf("                    [anim2] start\n");
+}
+void anim2_completed(lv_anim_t * a)
+{
+    LV_UNUSED(a);
```

Comment:
You can add global variables like `uint3t_t anim2_completed_called` and in `anim2_completed` just `anim2_completed_called++`.